config/default/package.json

```diff
@@ -21,7 +21,7 @@
     "chai-things": "^0.2.0",
     "eslint-plugin-json": "^2.0.1",
     "cht-conf": "^3.7.0",
-    "medic-conf-test-harness": "^2.1.3",
+    "cht-conf-test-harness": "^2.4.1",
```

Comment:
For some reason, the old version of the test harness refused to start on 22.04.  Not sure if there was something weird with Chrome/Puppeteer or what.  Updating to the newer version of cht-conf-test-harness needed to be done anyway and it solved the issue so the tests run fine on 22.04.